PyInstaller/loader/pyiboot01_bootstrap.py

```diff
@@ -111,6 +111,23 @@ def flush(*args):
 if sys.warnoptions:
     import warnings
 
+try:
+    import ctypes
+    import os
+    from ctypes import LibraryLoader, DEFAULT_MODE
+
+
+    class PyInstaller_CDLL(ctypes.CDLL):
+        def __init__(self, name, mode=DEFAULT_MODE, handle=None, use_errno=False, use_last_error=False):
```

Comment:
Should be `__init__(self, name, *args, **kwargs` since the only thing we care in `name`